packages/@okta/vuepress-site/docs/guides/oauth-for-okta-apis/use-client-credentials-grant-flow/index.md

```diff
@@ -0,0 +1,116 @@
+        }    
+    ]
+}
+```
+## Create a Service app
+Create an OAuth Service client app and register the public key with the Service app using the `/apps` API. The following example uses the **Add OAuth2 Client App** request in Postman:
+
+1. In Postman, select the **Add OAuth2 Client App** request within the **Apps** collection.
+2. On the **Body** tab, make the following noted changes to the parameter values and add the JWKS section.
+
+    For example:
+    ```json
+    {
+        "name": "oidc_client",
+        "label": "Name your app", //enter a name for the Service app
+        "signOnMode": "OPENID_CONNECT",
+        "credentials": {
+            "oauthClient": {
+                "token_endpoint_auth_method": "private_key_jwt" //update to private_key_jwt
+            }
+        },
+        "settings": {
+            "oauthClient": {
+                "client_uri": "http://localhost:8080",
+                "logo_uri": "http://developer.okta.com/assets/images/logo-new.png",
+                 "redirect_uris": [
+                    "http://{yourOktadomain}/callback"   
+                ],
+                "response_types": [
+                    "token" //remove all options except token
+                ],
+                "grant_types": [
+                    "client_credentials" //replace all options with client_credentials   
+                ],
+                "application_type": "service", //change to service
+                "jwks": { //add the JWKS section with the values from your generated jwks
+                    "keys": [
+                        {
+                            "kty": "RSA",
+                            "e":"AQAB",
+                            "kid": "name your jwks",
+                            "n":"AJncrKuine49_CE…..RL0HU="
+                        }    
+                    ]
+                }
+            }
+        }
+    }
+    ```
+
+   You can also create a service client using the dynamic `/oauth/v1/clients` registration endpoint:
+
+    ```json
+    { 
+    "client_name":"OAuthServiceClient",
+    "response_types":[ 
+        "token"
+    ],
+    "grant_types":[ 
+        "client_credentials"
+    ],
+    "token_endpoint_auth_method":"private_key_jwt",
+    "application_type":"service",
+    "jwks":{ 
+        "keys":[ 
+            { 
+                "kty":"RSA",
+                "e":"AQAB",
+                "use":"sig",
+                "kid":"7164109848619491",
+                "alg":"RS256",
+                "n":"1rM-gaSnogAzxdbuEc0M6nKkurrOdSXbSH_NqSjn-FDAWl4tZQHTg6cUaYkhsmpT5PAh5AngsCgvoqRydx-m6irfL-4xkvXV4PNbZkGpNdcQluEpwZvGwUz69wwo_tZl_3syBQSV3TRqRUv7-p7WyeHD78oM0SpPbm7cEzM_A2kDHROFiR9njWdr6R_PCHXmQHjp28ChwnxiA_WJczmmnU88K6cf0OatQsthn649XG2pqtSyZwr0uYu5mWiJvVAcGAD6eZH9u7hsGKQ3iJiPuToaY3QUqH-fJnU64_jxHIYlLdbgJhmY8LxeAomUdgYj3EegmQCtik3uWK00EnK3PQ"
+            }
+        ]
+    }
+    }'
+    ```
+ 
+3. When you finish, click **Send** to create the Service app with the JWKS.
```

Comment:
Is "send" correct here?

Reply:
It is if they are using Postman. The button says "Send".